### PR TITLE
feat: support qwik

### PR DESCRIPTION
**Pull request by Wyatex**
### Description
Add support for qwik.  
Related to #224

### CODE REVIEW

File: examples/qwik/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "@types/express": "4.17.13",
+    "@types/node": "latest",
+    "autoprefixer": "10.4.7",
+    "express": "4.17.3",
+    "node-fetch": "3.2.9",
+    "postcss": "8.4.14",
+    "tailwindcss": "3.0.24",
+    "typescript": "4.7.4",
+    "unplugin-icons": "workspace:*",
+    "vite": "3.0.2"
```
Comment (antfu) [inline examples/qwik/package.json:28]: Can you reduce the example to as minimal as possible? We probably don't need:

- debug related `--inspect-brk`
- eslint, typecheck
- tailwind, postcss
- express?
- assets like favicon.ico
- README

Comment (Wyatex) [reply examples/qwik/package.json:28]: I'm sorry I didn't notice these things. I'll clean them up. qwik requires express to provide services in production mode. But in production mode this plugin will generate some code such as `React.createElement`. I need some time to figure out how to correctly generate SSR code.

Comment (antfu) [reply examples/qwik/package.json:28]: We could keep express then 👍

**Comment by Wyatex** [comment]
I don't know how to inspect the compilation process. I tried the `vite-plugin-inspect` but it didn't work for qwik.  And I couldn't find another plugin to inspect the compilation process yet.

**Comment by Wyatex** [comment]
It seems that `React.createElement` was generated by esbuild. When I set `esbuild: { jsx: 'preserve'}` in `vite.config.ts`, vite told me:
```
Unexpected token (Note that you need plugins to import files that are not JavaScript)
```
Does this mean that the unplugin-icons process code after qwikVite?
But I don't understand why this is useful in DEV mode

File: examples/qwik/package.json (hunk 1 above)
Comment (Leizhenpeng) [reply examples/qwik/package.json]: The culprit may be svgr. I submitted a new version of code and now it perfectly supports qwik ~ 🚀~

**Pull request closed without merging**